pandas/_testing.py

```diff
@@ -608,10 +671,18 @@ def assert_index_equal(
         Specify comparison precision. Only used when check_exact is False.
         5 digits (False) or 3 digits (True) after decimal points are compared.
         If int, then specify the digits to compare.
+
+        .. deprecated:: 1.0.0
+           Use `rtol` and `atol` instead to define relative/absolute
+           tolerance, respectively. Similar to :func:`math.isclose`.
     check_exact : bool, default True
         Whether to compare number exactly.
     check_categorical : bool, default True
         Whether to compare internal Categorical exactly.
+    rtol : float, default 0.5e-5
```

Comment:
are these the same defaults as numpy?

Reply:
No, they are more in line with the previous behaviour of `check_less_precise`.

To guarantee that someone whose tests have `assert_almost_equal(a, b)` would still work. Numpy uses `rtol=1e-05`, while pandas's check_less_precise was closer to 0.5e-05. It's a small difference, but it made some pandas tests fail.

Is it OK to change to numpy's defaults? (I'd prefer that too)

Reply:
yes change to numpy default and adjust tests as needed by passing rtol/atol